chidioguejiofor/arfhimedes
# Setup repo

#### What does this PR achieve?
- adds config for flask, flask_restplus and SQLAlchemy
- initialises migrations

#### Manual Testing Steps
- Start server with python app.py
- Hit GET /api/sample
- you should see a sample response

#### Relevant Trello Tickets
- [Setup Repo](https://trello.com/c/OhIsRViE/12-setup-repo)
